packages/venia-concept/src/components/Pagination/pagination.js

```diff
@@ -134,4 +147,4 @@ class Pagination extends Component {
     };
 }
 
-export default classify(defaultClasses)(Pagination);
+export default withRouter(classify(defaultClasses)(Pagination));
```

Comment:
Try using the `compose` function from `redux` when wrapping a Component with more than one HOC.

```es6
import { compose } from "redux"

export default compose(
  withRouter,
  classify(defaultClasses)
)(Pagination)
```

Reply:
made the changes :+1: